keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,174 @@
+---
+title: doc-policies-for-third-party-content
+authors:
+  - "@aimeeu"
+  - "@jimangel"
+  - "@sftim"
+  - "@zacharysarah"
+owning-sig: sig-docs
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-10-20
+last-updated: 2019-10-20
+status: provisional
+---
+
+# doc-policies-for-third-party-content
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories](#user-stories)
+    - [Story 1 (fictional)](#story-1-fictional)
+    - [Story 2 (fictional)](#story-2-fictional)
+    - [Story 3 (actual)](#story-3-actual)
+    - [Story 4 (actual)](#story-4-actual)
+    - [Story 5 (actual)](#story-5-actual)
+    - [Story 6 (actual)](#story-6-actual)
+  - [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Graduation Criteria](#graduation-criteria)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+**Note:** This KEP does not target any release; SIG Docs follows a continuous release process for website content.
+
+## Summary
+
+This KEP seeks consensus on how Kubernetes docs handle two types of content:
+1. Content from third-party providers
+2. Content hosted on multiple sites ("dual-sourced content")
+
+Feature docs are not a place for vendor pitches. Nevertheless, SIG Docs receives pull requests to place advertising content on the Kubernetes website. Some PRs clearly do not belong in feature docs, but other instances are less clear.
+
+Feature docs also contain dual-sourced content. A good practice for code project docs is to host single-sourced content only, and to provide links to other providers’ single-sourced content. This simplifies version management and reduces the work required to maintain content.
+This KEP defines a policy on documentation content, so that authors can judge what is appropriate to propose and so that PR approvers can make consistent, fair decisions during the review process.
```

Comment:
It seems like SIG-Docs is already judging what PRs are "legitimate" and what PRs aren't. This KEP seeks to make that process transparent to the community.